rust/noosphere/src/sphere/builder.rs

```diff
@@ -166,6 +166,10 @@ impl SphereContextBuilder {
         let storage_path = self
             .storage_path
             .ok_or_else(|| anyhow!("No storage path configured!"))?;
+
+        #[cfg(not(target_arch = "wasm32"))]
+        std::fs::create_dir_all(&storage_path)?;
```

Comment:
I think we're reaching the long way around this problem here. Instead of avoiding creating a folder, would it be possible to check that a the key exists at the appropriate time?

Reply:
This was over-aggressive on my end, and not needed for any of our FS-backed Storages